cmd/podman/pull.go

```diff
@@ -61,7 +61,12 @@ func init() {
 
 // pullCmd gets the data from the command line and calls pullImage
 // to copy an image from a registry to a local machine
-func pullCmd(c *cliconfig.PullValues) error {
+func pullCmd(c *cliconfig.PullValues) (retError error) {
+	defer func() {
+		if retError != nil && exitCode == 0 {
+			exitCode = 1
```

Comment:
ExitCode should be 125

Reply:
We should make this a constant and start using it.

PODMANFAILED or something.

Reply:
I get 125:

```
$ bin/podman pull fooo/bar ; echo $?
Trying to pull docker://fooo/bar...ERRO[0001] Error pulling image ref //fooo/bar:latest: Error determining manifest MIME type for docker://fooo/bar:latest: Error reading manifest latest in docker.io/fooo/bar: errors:
denied: requested access to the resource is denied
unauthorized: authentication required
 
Failed
ERRO[0001] error pulling image "docker://fooo/bar"      
Error: error pulling image "fooo/bar": Invalid image name "fooo/bar", expected colon-separated transport:reference
125

```